src/test/scala/chiselTests/RecordSpec.scala

```diff
@@ -146,6 +146,53 @@ class RecordSpec extends ChiselFlatSpec with RecordSpecUtils with Utils {
     e.getMessage should include("contains aliased fields named (bar,foo)")
   }
 
+  they should "be OpaqueType for single element maps" in {
+    class SingleElementRecord extends Record {
+      private val underlying = UInt(8.W)
+      val elements = SeqMap("" -> underlying)
```

Comment:
is it important that the string be empty? What if I put ("unused" -> underlying)? Can we write a test for that case?

Reply:
The name should be required to be empty, I've added this to my comments